pandas/core/groupby/groupby.py

```diff
@@ -5720,6 +5724,113 @@ def sample(
         sampled_indices = np.concatenate(sampled_indices)
         return self._selected_obj.take(sampled_indices, axis=self.axis)
 
+    def _idxmax_idxmin(
+        self,
+        how: Literal["idxmax", "idxmin"],
+        ignore_unobserved: bool = False,
+        axis: Axis | None | lib.NoDefault = lib.no_default,
+        skipna: bool = True,
+        numeric_only: bool = False,
+    ):
+        """Compute idxmax/idxmin.
+
+        Parameters
+        ----------
+        how: {"idxmin", "idxmax"}
```

Comment:
nitpick i think missing space between "how" and colon

Reply:
also usually but not always we have single-quotes inside these (no idea why and i genuinely dont care, extra since this is private)

Reply:
Thanks - fixed in #54234